missing_numbers.py

```diff
@@ -0,0 +1,14 @@
+def missing_number(b):
+    if len(b) != 0:
+        first_list = []
+        for x in range(b[0], b[-1]+1):
```

Comment:
You can generate a new list without looping
```py
my_cute_list = [*range(2, 7)]
```

Reply:
Alright, have just tried it out and its just cool